interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
I'd say that the dodgyness derived from two things:

1. The process of downloading and installing the large volume of packages into `node_modules` is fragile, which seems to be attributable to `npm` itself — builds on my dev instance rarely have trouble, but the AWS deployment, which must re-create that cache every time, will most commonly fail with an error downloading or installing something into `node_modules`.
2. Several `npm` packages from our toolchain are old and crufty. We need to go through and bring our toolchain into 2017. That process will be easier to handle once we have a robustly Dockerized interface build.